### PR TITLE
refactor(notice): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
# My first tailwind encounter!

Supports #1628 #1500

There are some I didn't know how to convert to tailwind. I have TODO or FIXME on top of the ones I need some guidance on.

Also, for the space between title/message/link: I used `text-#h`. However, the space is not much compared to 
Figma:
image

Implementation (`with text-#h`):
image

If I just have `text-#` and add `mb-1` or `.mt-1`  (`m*-2` is too much space) it looks closer to Figma render:
image

cc'ing @macandcheese @bstifle

### CODE REVIEW

**Comment by julio8a** [comment]
Still have the commit history from master. Will just work from the main repo for future work. I did get a branch into this repo for screener.

**Comment by eriklharper** [comment]
> Still have the commit history from master. Will just work from the main repo for future work. I did get a branch into this repo for screener.

Weird.  Not sure why that big set of commits is there.  Not a huge deal since it gets squashed.

**Comment by eriklharper** [comment]
Would be great for @bstifle to review the Screener diffs to see if they look acceptable.  Looks like the text on stacked UI like Notice looks a bit tighter, with less spacing than before.

File: src/components/calcite-notice/calcite-notice.scss
Comment (jrowlingson) [inline src/components/calcite-notice/calcite-notice.scss:49]: `width[1/2]`

**Comment by bstifle** [comment]
please see latest on figma. some spacing was bugged and i fixed it https://www.figma.com/file/E3SB0i5wPy7AagB7KcjF05/?node-id=0%3A13185

**Comment by eriklharper** [comment]
> please see latest on figma. some spacing was bugged and i fixed it https://www.figma.com/file/E3SB0i5wPy7AagB7KcjF05/?node-id=0%3A13185

I'm seeing this when I click that link:
image

**Comment by paulcpederson** [comment]
@eriklharper  I think you need to sign in with your esri email and the design folks will need to invite you?

File: src/components/calcite-notice/calcite-notice.scss
Comment (julio8a) [reply src/components/calcite-notice/calcite-notice.scss]: This has been added!

**Comment by julio8a** [comment]
@asangma @jrowlingson Any more feedback on this?

@bstifle reviewed the branch and have me his blessing.

**Review by bstifle — APPROVED** [review]

**Comment by julio8a** [comment]
@asangma @jrowlingson

any feedback on this before I merge?

**Comment by paulcpederson** [comment]
@julio8a @bstifle you should check the screener and accept this change if it looks good, then we can merge.

**Comment by julio8a** [comment]
Will do

**Comment by bstifle** [comment]
no way i can check it after its been approved great UX screener GREAT

**Comment by julio8a** [comment]
image

**Comment by julio8a** [comment]
@eriklharper is there a way to still see what changed in screener after it's been approved?

**Comment by eriklharper** [comment]
> @eriklharper is there a way to still see what changed in screener after it's been approved?

Unfortunately, there is not a way to do that once the changes are approved, AFAIK, but the one thing you can do is check out the history of the story, which is accessible in the sidebar here:

image

You'll have to save two images in the history, or open them in new tabs and then flip back and forth quickly to see the changes.

**Pull request merged**
